src/codegen.cpp

```diff
@@ -1974,6 +1974,7 @@ class jl_codectx_t {
 
     Value *pgcstack = NULL;
     Instruction *topalloca = NULL;
+    Value *world_age_value = NULL;
```

Comment:
```suggestion
    Value *world_age_on_entry = NULL;
```

This name makes it a bit more likely folks will think about whether the world age is constant in the particular block of code being generated